src/components/orders/AmountsDisplay/index.tsx

```diff
@@ -5,30 +5,32 @@ import { TokenErc20 } from '@gnosis.pm/dex-js'
 
 import { Order } from 'api/operator'
 
+import { Network } from 'types'
+
+import { useNetworkId } from 'state/network'
+
 import { formatSmartMaxPrecision } from 'utils'
 
-import { BlockExplorerLink } from 'apps/explorer/components/common/BlockExplorerLink'
+import { TokenDisplay } from 'components/common/TokenDisplay'
 
-import { RowContents, RowTitle, StyledImg, /*UsdAmount,*/ Wrapper } from './styled'
+import { RowContents, RowTitle, /*UsdAmount,*/ Wrapper } from './styled'
```

Comment:
delete the commentedboi?

Reply:
Nope, that's there because we WILL have usd estimations some day :crossed_fingers: